src/components/header/header.three.js

```diff
@@ -11,12 +11,15 @@ import Avatar from '@material-ui/core/Avatar';
 import { Query } from 'react-apollo';
 import { useUser } from '../../store/user-context';
 import gql from 'graphql-tag';
+import CreateCompanyScreen from '../../apps/admin/routes/company/index';
 
 export function HeaderLeft({ sections, classes, ...props }) {
+  const [showCompanyScreen, toggleCompanyScreen] = useState(false);
   const { data, logout } = useUser();
-  console.log('DATA IN HEADER THREE:', data);
   const { user } = data;
 
+  const isUserLoggedIn = user ? true : false;
```

Comment:
Why not use the same `user`-based implementation as already contained herein (below)?